src/engine/execute.js

```diff
@@ -197,16 +198,21 @@ var execute = function (sequencer, thread) {
         getParam: function (paramName) {
             return thread.getParam(paramName);
         },
-        startHats: function(requestedHat, opt_matchFields, opt_target) {
+        startHats: function (requestedHat, optMatchFields, optTarget) {
             return (
-                runtime.startHats(requestedHat, opt_matchFields, opt_target)
+                runtime.startHats(requestedHat, optMatchFields, optTarget)
             );
         },
         ioQuery: function (device, func, args) {
             // Find the I/O device and execute the query/function call.
             if (runtime.ioDevices[device] && runtime.ioDevices[device][func]) {
                 var devObject = runtime.ioDevices[device];
-                return devObject[func].call(devObject, args);
+                // @todo Figure out why eslint complains about no-useless-call
+                // no-useless-call can't tell if the call is useless for dynamic
+                // expressions... or something. Not exactly sure why it
+                // complains here.
+                // eslint-disable-next-line no-useless-call
+                return devObject[func].call(devObject[func], args);
```

Comment:
This is different from the old code.
Also, I wonder if this would make the code easier to understand and/or suppress the `eslint` complaint:
```js
var realFunc = devObject[func];
realFunc.call(devObject, args);
```
(Please double-check that my suggestion is equivalent to the original code...!)